challenges/advanced-variadic-generics/question.py

```diff
@@ -0,0 +1,25 @@
+"""
+TODO:
+
+Define a `Array` type that supports addition of arrays with the same shape.
```

Comment:
"Define an `Array` type that supports element-wise addition of arrays with identical dimensions and types."

Same for solution.